src/ui.form-calendar/index.vue

```diff
@@ -252,6 +252,7 @@ const props = defineProps({
 
   /**
    * Min date in format date string or Date.
+   * Provide dateFormat props to use string format.
```

Comment:
Min date in JS Date Objects or Date String formatted in provided `props.dateFormat`.